src/liballoc/arc.rs

```diff
@@ -577,9 +585,7 @@ impl<T: ?Sized> Drop for Arc<T> {
         // [1]: (www.boost.org/doc/libs/1_55_0/doc/html/atomic/usage_examples.html)
         atomic::fence(Acquire);
 
-        unsafe {
-            self.drop_slow()
-        }
+        unsafe { self.drop_slow() }
```

Comment:
We allow one-line unsafe blocks if there is a single expression, the previous one is a single statement. This is obviously an imperfect heuristic, foiled here because there is no semi-colon, even though we don't use the value. We don't have type info in rustfmt (for now), so can't insert the semicolon. I'll fix this instance manually.